ladybug_geometry/geometry2d/line.py

```diff
@@ -71,12 +76,24 @@ def length(self):
 
     def to_array(self):
         """ A nested list representing the two line endpoint coordinates.
-            Returns: 
+            Returns:
                 Nested tuples ((pt1.x, pt1.y), (pt2.x, pt2.y)), where
                 pt1 and pt2 represent the endpoints of the line segment.
         """
         return (self.p1.to_array(), self.p2.to_array())
 
+    @classmethod
+    def from_array(cls, line_array):
+        """ Consumes a nested list representing the two line endpoint coordinates
+        and returns a LineSegment2D.
```

Comment:
Can we make this one line like so:
```
Create a LineSegment2D from a nested array of two endpoint coordinates.
```
As a general note, I try to use the work "array" when something can be either a list or tuple.

Also, I think you need to have a full space between this line and the `Args` for it to be parsed correctly by Sphinx for the online docs.